model/metrics/system-metrics.yaml

```diff
@@ -361,22 +363,31 @@ groups:
       - ref: system.filesystem.mode
       - ref: system.filesystem.mountpoint
 
-  # system.network.* metrics and attribute group
-  - id: attributes.system.network
-    prefix: system.network
+  # system.network.* metrics and attribute group  
+  
+  # General network attributes
+  - id: attributes.network
```

Comment:
what do you think about moving this attribute to the attributes registry within this PR (into the network.yaml file)?

Same for `disk.io.direction`

Reply:
It makes more sense to define those 2 in context specific files yes. We don't have a `disk.yaml` file but I guess we can create one.

Then the attributes could be referenced from `system` and `process` metrics (and even at https://github.com/open-telemetry/semantic-conventions/pull/282#issuecomment-1757735366 :))

Reply:
That was what I was grappling with; I don't know if there is a plan for `disk` or `network` namespaces for metrics, so I didn't want to make those files in the `metrics` folder.

Is there a more general global attribute registry where these could go instead? I would prefer to take them out of this file if that's the case.

Reply:
The attributes' registry is defined at https://github.com/open-telemetry/semantic-conventions/tree/main/model/registry (and https://github.com/open-telemetry/semantic-conventions/tree/main/docs/attributes-registry).

Reply:
I moved these attributes to the global registry. I put `network.io.direction` in `network.yaml` and `disk.io.direction` in a new `disk.yaml` file.